pages/pricing.tsx

```diff
@@ -57,6 +59,23 @@ const Pricing = () => {
 	const pricingStartDate = new Date(2023, 6, 1); // 1st July 2023
 	const today = new Date();
 
+	React.useEffect(() => {
+		// tracking events on every button clicked 
+		const localStorageAnonymousId = localStorage.getItem('AnonymousId');
+		const anonymousId: string = (localStorageAnonymousId && localStorageAnonymousId != null) ? localStorageAnonymousId : uuidv4();
+		rudderEventMethods().then((response) => {
+			response?.track("", "pricing-page", { "isYearly": isYearly }, anonymousId);
+		});
+		localStorage.setItem('AnonymousId', anonymousId);
+
+	}, [isYearly]);
+
+	React.useEffect(() => {
+		rudderEventMethods().then((response) => {
+			response?.track("", "pricing page called", { eventStatusFlag: 1 }, "anonymous") //Anonymous Id is set in local storage as soon as the user lands on the webiste.
+		});
+	}, []);
+
```

Comment:
Requires manual testing

Reply:
Can see the data in rudderstack, data takes time to gte ingested into bigquery. Plus, @ajayVibinex, please make sure that same page events should be named same. `Pricing-page` can be a good option here.